src/EventStore.Core/ClusterVNode.cs

```diff
@@ -1679,9 +1681,29 @@ public class ClusterVNode<TStreamId> :
 					_mainQueue.Publish(new AuthenticationMessage.AuthenticationProviderInitializationFailed());
 				} else {
 					_mainQueue.Publish(new AuthenticationMessage.AuthenticationProviderInitialized());
+					_mainQueue.Publish(TimerMessage.Schedule.Create(TimeSpan.FromHours(24),
+						new PublishEnvelope(_mainQueue),
+						new SystemMessage.VerifyNodeCertificateExpiry()));
+
 				}
 			});
 		}
+		
+		public void Handle(SystemMessage.VerifyNodeCertificateExpiry message) {
+			var (certificate, intermediates) = _options.LoadNodeCertificate();
+			
+			var certExpiryDate = certificate.NotAfter;
+			var addMonthToCurrentDate = DateTime.Now.AddDays(30);
+
+			if (addMonthToCurrentDate >= certExpiryDate) {
+				Log.Warning(
+					$"Certificates are going to expire within 30 days. Current Date: {DateTime.Now}, Expiry Date: {certificate.NotAfter}");
```

Comment:
You should use the positional formatting for Serilog rather than string interpolation here:
```suggestion
					"Certificates are going to expire within 30 days. Current Date: {currentDate}, Expiry Date: {certificateExpiryDate}", DateTime.Now, certificate.NotAfter);
```